roop/utilities.py

```diff
@@ -54,7 +72,8 @@ def restore_audio(target_path: str, output_path: str) -> None:
 
 def get_temp_frame_paths(target_path: str) -> List[str]:
     temp_directory_path = get_temp_directory_path(target_path)
-    return glob.glob(os.path.join(temp_directory_path, '*.png'))
+    return [os.path.join(temp_directory_path, file) for file in os.listdir(temp_directory_path) if
```

Comment:
can we simplify this condition using any? I can barely understand this

Reply:
done